src/common/debug.c

```diff
@@ -55,7 +55,8 @@ debug_fini_locked(void)
 
 	rc = D_LOG_DEREGISTER_DB(DAOS_FOREACH_DB);
 	if (rc != 0) /* Just print a message but no need to fail */
-		D_PRINT_ERR("Failed to deallocate daos debug bits: %d\n", rc);
+		D_PRINT_ERR("Failed to deallocate daos debug bits: "DF_RC"\n",
```

Comment:
this error message is redundant as the debug macro prints an error

Reply:
right, if you dive deep enough to see what's inside the macro:

#define _D_LOG_DEALLOCATE_DBG_BIT(dbgbit, s_name, l_name, mask, rc)     \
        rc = d_log_dbg_bit_dealloc(#s_name);                            \
        if (rc < 0)     {                                               \
                rc = -DER_UNINIT;                                       \
                D_PRINT_ERR("Could not free debug bit " #s_name "\n");  \
                break;                                                  \
        }